syftlib/src/main/java/org/openmined/syft/networking/datamodels/syft/SpeedTest.kt

```diff
@@ -0,0 +1,11 @@
+package org.openmined.syft.networking.datamodels.syft
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.openmined.syft.networking.datamodels.NetworkModels
+
+@Serializable
+class SpeedTest(
```

Comment:
Maybe `SpeedCheck`? I am thinking we tend to use the suffix `Test` for tests.
Not a big deal though.